client/src/main/java/org/apache/rocketmq/client/producer/LocalTransactionExecutor.java

```diff
@@ -18,6 +18,6 @@
 
 import org.apache.rocketmq.common.message.Message;
 
-public interface LocalTransactionExecuter {
-    public LocalTransactionState executeLocalTransactionBranch(final Message msg, final Object arg);
+public interface LocalTransactionExecutor {
```

Comment:
For cinfiguration issue, we can fix them by pointing them to the same right spelling config.

    private int rightConfig;
       @Deprecated    
    public int getTypoConfig(){
         return rightConfig;
    }

    @Deprecated
    public void setTypoConfig(int val) {
        this.rightConfig = val;
    }

    getter/setter for right config...


But since client config and server config is working in tatolly different ways that server config uses reflection to load configurations so this should be only work for client configuration typo issue. 

I suggest we post another two typo issues for client configuration and server configuration.